_source/_posts/2022-05-15-jhipster-ionic-spring-boot.adoc

```diff
@@ -0,0 +1,481 @@
+---
+layout: blog_post
+title: "Integrate Ionic and Spring Boot Securely with JWT and OAuth 2.0"
+author: matt-raible
+by: advocate
+communities: [mobile,java,javascript]
+description: "Build an Ionic frontend for your Spring Boot backend, all using the power of JHipster!"
+tags: [ionic, spring-boot, spring-security, jhipster]
+tweets:
+- "Create an Ionic app with a Spring Boot backend in minutes with the new Ionic blueprint for @jhipster!"
+- "Introducing JHipster Ionic: 'Wow! Did you really just build that?!'"
+- "Ionic is awesome. Learn how you to create apps quickly using this powerful framework. 💪"
+image:
+type: conversion
+github: https://github.com/oktadev/okta-jhipster-ionic-example
+---
+:page-liquid:
+:toc: macro
+:experimental:
+:commandkey: &#8984;
+
+[comment]
+--
+todo: research and figure out best title.
+Other options:
+
+- Integrate Ionic and Spring Boot with OAuth/OIDC
+- Integrate Ionic and Spring Boot
```

Comment:
Integrate Ionic and Spring Boot with the New JHipster Blueprint - 57 neutral 
Integrate Ionic and Spring Boot - 47 negative